pkg/plugins/golang/v3/api.go

```diff
@@ -59,6 +59,8 @@ type createAPISubcommand struct {
 
 	// runMake indicates whether to run make or not after scaffolding APIs
 	runMake bool
+
+	useWorkspaces bool
```

Comment:
@jakobmoellersap. 

Since the workspace option changes the behaviour and layout then, I think it should be like the multi-group one. That means if we call `kubebuilder edit --workspaces=true` we will store the value in the PROJECT file (which represents the config of the project). Therefore, when we read the config we would have this information.

See: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4-multigroup/PROJECT#L4
And its docs: https://book.kubebuilder.io/migration/multi-group.html